docs/config_guide.md

```diff
@@ -0,0 +1,24 @@
+# Configuration Guide
+Currently all configuration is done via a single git-ignored configuration file in the project root.
+
+To create a project configuration, create a file named `config.env` in the project root. Populate it with `=`-separated key-value pairs. Ensure at least the following parameters are provided:
+
+```sh
+# Used to sign JWT for community business (CB) users
```

Comment:
@astroash @RogeredBacon this was missed in the review. `SECRET` is used to key the HMAC operation when storing passwords. See #213